CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.1] 2021-12-15
```

Comment:
I think this should be a minor instead of a patch, provided it's an enhancement rather than a fix.